include/fmt/ranges.h

```diff
@@ -426,6 +433,7 @@ struct range_formatter<
   }
 
  public:
+  bool string_or_debug_string = false;
```

Comment:
This should be private and I suggest renaming to just `string_` since debug string is just a variation.